capstone/src/main/resources/static/profile.html

```diff
@@ -55,7 +56,7 @@
             <label>Last Name:</label>
             <input type="text" name="lastname" id="ind-lastname" placeholder="Your last name.."></input>
             <label>University:</label>
-            <p id="university-form-display"></p>
+            <p id="university-form-display" style="display:none"></p>
```

Comment:
Correct me if I'm wrong, but I think this is the line you were talking about in my other PR. I think its fine if you hide it and then reveal it in the js function.